tests/requirements.txt

```diff
@@ -16,5 +16,5 @@ coverage>=4.5.2
 nltk>=3.2
 # spacy>=2.0.18
 pandas>=0.20
-umap-learn==0.3
-numba==0.42
+umap-learn>=0.3.9
```

Comment:
Would it be easier to fix the version to ==0.3.9 and periodically bump it so we don't get surprise cascading failures for the next numba dep change?

Reply:
It  would!

Reply:
@bbengfort if we go the `==` route, would you recommend  we keep #790 open  (and  rename?)  to remind  us  to do the periodic  bumps?